### DISCUSSION TITLE
does laragon have backup folder for MySQL like Xampp or am i not safe?

### DISCUSSION THREAD

**Discussion by amirhamdy450** [leokhoa/laragon#907] [Q&A]
i switched from **XAMPP** to **laragon** because the MySQL fails 2 times a week no matter what i do and i keep getting inconsistent errors that suggest systematic corruption of the mysql and despite trying trying to reinstall XAMPP before, it seems like no matter what i do the problem arises after a while no matter how long MySQL stays stable it inevitably fails, i can drop my error logs if anyone is interested or have any knowledge of this peculiar behavior,  anyways i don't if it makes a difference but i switched to laragon and it has cleaner interface and felt more simpler however one thing i noticed in the MySQL folder is that there is no backup folder,  is this normal?

**Comment by AWqxKAWERbXo** [comment]
I guess, there is none.

That said: Laragon never failed and corrupted my database. That's the reason I switched, too.

Also that said: I do a full hard-disk backup once a week and my CMS does db backups once an hour.

You can manually backup MySQL by HeidiSQL which is delivered with Laragon. That's a few clicks if you're not familiar with terminals and stuff.

Or you can set up a cron with cronical which is also part of Laragon:

![image](https://github.com/user-attachments/assets/481748a0-9b05-48ed-8974-5fbc30573715)

I never used it but feel free to write a short tutorial that will backup databases automatically in a seperated folder.

https://github.com/leokhoa/laragon/discussions/882
https://pen-y-fan.github.io/2023/01/28/how-to-run-cron-jobs-on-windows/
https://github.com/leokhoa/laragon/discussions/356
https://github.com/mgefvert/Cronical
